### PR TITLE
Feature/versioning

### PR DESCRIPTION
**Pull request by evgeniilezhnin**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/versioning/CommitDAORdbImpl.java [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+                        new ModelDBException(
+                            "Couldn't find head commit by sha : " + request.getCommitHead(),
+                            Code.NOT_FOUND));
+        Long headTime = headCommitEntity.getDate_created();
+        commitQueryBuilder.append(" AND cm.date_created <= " + headTime);
+      }
+
+      Query<CommitEntity> commitEntityQuery =
+          session.createQuery(
+              commitQueryBuilder.append(" ORDER BY cm.date_created ASC").toString());
```
Comment (evgeniilezhnin) [inline backend/src/main/java/ai/verta/modeldb/versioning/CommitDAORdbImpl.java]:
```suggestion
              commitQueryBuilder.append(" ORDER BY cm.date_created DESC").toString());
```
It should be in reverse order to get the last created commit by:
commits?pagination.page_number=1&pagination.page_limit=1
Please confirm

Comment (evgeniilezhnin) [reply backend/src/main/java/ai/verta/modeldb/versioning/CommitDAORdbImpl.java]: @ravishetye @ad-47

**Comment by evgeniilezhnin** [comment]
LGTM

**Comment by evgeniilezhnin** [comment]
> LGTM

If we'll resolve this: https://github.com/VertaAI/modeldb/pull/176#discussion_r386852491

**Comment by ravishetye** [comment]
let's treat this as an overall feature branch and not rush into merging it this week.
cc @conradoverta @evgeniilezhnin @ad-47

**Pull request merged**